engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdown.java

```diff
@@ -39,7 +39,7 @@
     private static final String LIST = "list";
     private static final String LIST_ITEM = "list-item";
 
-    private Binding<List<T>> options = new DefaultBinding<List<T>>(Lists.<T>newArrayList());
+    private Binding<List<T>> options = new DefaultBinding<>(Lists.<T>newArrayList());
```

Comment:
Using `new ArrayList<>()` should be unambiguous and work even without the type specifier.